Sources/Public/CalendarView.swift

```diff
@@ -269,6 +269,12 @@ public final class CalendarView: UIView {
       anchorLayoutItem = nil
     }
 
+    if content.monthsLayout.isPaginationEnabled {
+      scrollView.decelerationRate = .fast
+    } else {
+      scrollView.decelerationRate = .normal
+    }
```

Comment:
Is this something that may be beneficial to also have configurable within the horizontal options?

Reply:
I considered this (and it's certainly something we could add if there's enough demand), but after trying it, it's honestly really weird UX - unlike any horizontally-paginating control I've ever used lol. For the sake of keeping the API simple, I'm inclined to hold off on adding this extra option for now.

Reply:
Makes sense to me!